devices_test.go

```diff
@@ -67,3 +70,40 @@ func TestCompareStrStrMap(t *testing.T) {
 		t.Error("Maps with different content are OK!")
 	}
 }
+
+func TestUpdateAddressableWhenAddressableIsExist(t *testing.T) {
+	svc = &Service{}
+	svc.ac = mock.AddressableClientMock{}
+	addressable := models.Addressable{Name: "addressable-device"}
+
+	addr, err := updateAddressable(&addressable)
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+	if addr.Id.Hex() == "" {
+		t.Fatal("It need return addressable with Id.")
```

Comment:
"It need return addressable with Id." -> "Addressable should be returned with Id"